### DISCUSSION TITLE
-S Module Problem with multiphase FEPX simulation

### DISCUSSION THREAD

**Discussion by mehbah** [neperfepx/neper#163] [Q&A]
Hi,
When I'm running "neper -S FepxResults", I take the following error. Which file can not be read? 
Kind Regards,
Mehran

```
========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.1.1
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt
Info   : Running on 8 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2020, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/mehran/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -S loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] FepxResults
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   :   - Reading arguments...
Info   : Writing simulation directory from FEPX raw result directory...
Info   :     [o] Writing directory `FepxResults.sim'...
Info   :   - Parsing FEPX results...
var = 12
Error  :     > Failed to read file.
Aborted (core dumped)
```

**Comment by rquey** [comment]
There is way too few information here. Please make it so that I can reproduce your bug, and also specify what is your FEPX version.

**Reply by mehbah** [reply]
Hi rquey!

I have constructed the microstructure and mesh file with the following commands,

neper -T -n 100 -domain "cube(100,200,300)" -morpho "diameq:0.2*lognormal(1,0.35)+0.8*lognormal(2,0.6),1-sphericity:lognormal(0.145,0.03)" -group mode -o simulation -reg 1

neper -M simulation.tess -order 2 

I have also attached the .msh, simulation.config and FEPX raw data files. I used FEPX 1.1.1.

[FepxResults.zip](https://github.com/rquey/neper/files/6249128/FepxResults.zip)

Kind regards,
Mehran

**Reply by rquey** [reply]
Neper -S does not read multiphase FEPX results yet, but you are in luck: since both phases are FCC and have 12 slip systems, you can work around this by removing the second "12" on the `number_of_slip_systems` line in `post.report`, and Neper -S will take it.

We'll fix this shortly.
